ocrd_models/ocrd_models/ocrd_mets.py

```diff
@@ -210,8 +210,8 @@ def file_groups(self):
         """
 
         # WARNING: Actually we cannot return strings in place of elements!
```

Comment:
As the warning warns, we cannot just return a list of strings and use it as a list of fileGrp memory objects. However, I am also a bit confused because all tests seem to pass successfully which was not the case when I first implemented the caching mechanism. It should rather throw an exception saying that the string object does not support a call of a fileGrp method.